admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node can reserve a portion of its resources for use by all underlying
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node
+components] (e.g., kubelet, kube-proxy, Docker) and the remaining system
+components (e.g., *sshd*, *NetworkManager*) on a node host. Once specified, the
+scheduler has more information about the resources (e.g., memory, CPU) a node
```

Comment:
How about substituting "resources (...)" with a link to the Developer Guide "Compute Resources" topic (maintaining "resources" as the link text)?